ferrocene/ci/docker-images/aarch64-ubuntu-20/Dockerfile

```diff
@@ -0,0 +1,128 @@
+# SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
I wonder how much we can reduce duplication here with the x86_64 Dockerfile? Can we get to a point where we have a single Dockerfile for the two images?

Reply:
I was looking a bit at that last day and I think the main issue is checksums, which is a surmountable problem.

Reply:
I'm kind of focusing on this path in https://github.com/ferrocene/ferrocene/pull/690, unless you think I should prioritize getting it into this PR.

Reply:
Let's do it in a followup PR.